www/_layouts/base.html

```diff
@@ -3,12 +3,27 @@
 ---
 
 <!DOCTYPE html>
-<html lang="{% if page.language %}{{ page.language }}{% else %}{{ site.language }}{% endif %}">
+{% if page.language %}
+    {% assign language = page.language %}
+{% else %}
+    {% assign language = site.language %}
+{% endif %}
+
+<html lang="{{ language }}">
 {% include head.html %}
 <body>
     {{ content }}
     {% include analytics.html %}
     <script defer type="text/javascript" src="{{ site.baseurl }}/static/js/index.js"></script>
     <script defer type="text/javascript" src="{{ site.baseurl }}/static/js/twitter.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.js"></script>
+	<script type="text/javascript">
+		docsearch({
+		    apiKey: '0a916ab198bd93d031aa70611271e42e',
+		    indexName: 'cordova',
+		    inputSelector: '#header-search-field',
+		    algoliaOptions: { 'facetFilters': [ "version:6.x","language: {{ language }}" ] }
```

Comment:
It is the same language value that is used. I think it is ok with defining it on top...